frontend/public/components/storage/attach-pvc-storage.tsx

```diff
@@ -0,0 +1,436 @@
+import * as _ from 'lodash-es';
```

Comment:
Moved `attach-storage.tsx` to `attach-pvc-storage.tsx`. Shows as a new file because `attach-storage.tsx` was then used to create the parent component which accepts extensions.